wiki/Disambiguation/en.md

```diff
@@ -5,6 +5,6 @@ Disambiguation in the osu! wiki is the process of resolving conflicts that arise
 ## List of disambiguation articles
 
 - [Bancho](/wiki/Bancho)
-- [Easy](/wiki/Easy)
+- Easy
```

Comment:
The link here should not be removed. It is pointing to a redirect, but should really link to the page underneath it.

Reply:
goop

is this the only one that's referring to disambig (did you check all of them) ?

Reply:
Just checked again but yeah, it looks like this is the only one that actually intends to link to the disambiguation page while all of the others were linking to it by mistake.